content-src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -17,26 +25,22 @@ const ActivityFeedItem = React.createClass({
       </div>
     </li>);
   }
-});
+}
 
-ActivityFeedItem.propTypes = {
-  url: React.PropTypes.string.isRequired,
-  image: React.PropTypes.string, // TODO: Have fallback for this, change to icon
-  title: React.PropTypes.string.isRequired
-};
+class ActivityFeed extends React.Component {
+  static get propTypes() {
+    return {
+      sites: React.PropTypes.array.isRequired
+    };
+  }
 
-const ActivityFeed = React.createClass({
   render() {
-    const {props} = this;
+    const props = this.props;
```

Comment:
Tweaked this for consistency with the rest of the codebase. I think this was the only instance i saw where we destructured.